build.sbt

```diff
@@ -122,7 +122,7 @@ lazy val slick = Project("slick", file("slick")).
   settings(commonSettings: _*)
 
 lazy val spark = Project("spark", file("spark")).
-  dependsOn(util, raster).
+  dependsOn(util, raster, rasterTestkit).
```

Comment:
This will pull test kit dependency for the main build, we shouldn't do that. I believe just doing `rasterTestkit% "test"` should scope it down to tests only.